metafora.go

```diff
@@ -442,12 +431,12 @@ func (c *Consumer) stopTask(taskID string) bool {
 	if !ok {
 		// This can happen if a task completes during Balance() and is not an error.
 		c.logger.Log(LogLevelWarn, "Tried to release a non-running task: %s", taskID)
-		return false
+		return
 	}
 
 	// all handler methods must be wrapped in a recover to prevent a misbehaving
 	// handler from crashing the entire consumer
```

Comment:
Oops, good catch! I actually meant to *move* the individual go calls here, but instead I duplicated them! Will remove the outer `go`s.